src/serpentTools/parsers/depmatrix.py

```diff
@@ -92,9 +95,16 @@ def _read(self):
         with open(self.filePath) as stream:
             line = stream.readline()
             self.deltaT = float(line.split()[-1][:-1])
+            line = stream.readline()
+
+            if "flx" in line:
+              self.flx = float(line.split()[-1][:-1])
+              line = stream.readline()
+
+            if "zeros" in line:
+              line = stream.readline()
```

Comment:
Why is the zero line added? If that's also new that's one thing but I'm pretty sure this line existed in earlier versions of Serpent.

Related to the removal of the zero line down below?